index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./styles.css">
+    <script src="./script.js"></script>
+</head>
+
+<body>
+    <header class="header">
+        <img src="./images/logo.svg" alt="" class="logo">
+        <div class="navbarContainer">
+            <div class="navbar">
+                <a href="#home">Home</a>
+                <a href="#about">About</a>
+                <a href="#store">Store</a>
+            </div>
+            <div class="info">
+                <div class="contact icon">
+                    <i class="fa fa-phone"></i> + 123 456 789
+                </div>
+                <div class="basket" id="basket" onclick="openBasket()">
+                    <i class="fa fa-shopping-cart"></i>
+                    <span id="itemsCount">0</span> items - $<span id="itemsTotalPrice">0</span>
+                </div>
+            </div>
+        </div>
+        <span class="collapseMenuIcon" onclick="miniMenu()">
+            <i class="fa fa-bars"></i>
+        </span>
+    </header>
+    <main>
+        <section class="home">
+            <div class="basketContainer">
+                <div class="basketItems">
+                </div>
+                <div class="total">
+                    <span>Total</span>
+                    <span id="totalPriceBasket">$15</span>
+                </div>
+                <div class="basketButtons">
+                    <button class="button">CLEAR CART</button>
+                    <button class="button button-pink">CHECKOUT</button>
+                </div>
+            </div>
+            <div class="content">
+                <div class="whiteText title">Welcome To <span class="titleStar font-56">Grandma's</span>
+                </div>
+                <button class="button" style="font-size: x-large;">EXPLORE</button>
```

Comment:
Please add the style in the css file
```suggestion
                <button class="button">EXPLORE</button>
```